2048/2048/src/main/java/com/appcoins/eskills2048/repository/StatisticsRepository.java

```diff
@@ -14,7 +17,9 @@
   }
 
   public Single<GeneralPlayerStatsResponse> getUserStatistics(String applicationId,
-      String userWalletAddress, StatisticsTimeFrame timeFrame) {
-    return api.getGeneralPlayerStats(applicationId, userWalletAddress, 3, 2, timeFrame.name());
+      String userWalletAddress, MatchDetails.Environment matchEnvironment,
+      StatisticsTimeFrame timeFrame) {
+    return api.getGeneralPlayerStats(applicationId, userWalletAddress, null, 3, 2, matchEnvironment,
```

Comment:
wait... why are you sending a null? shouldn't `product` be optional on the server side?

Reply:
It is, on the server side. It is not on the method getGeneralPlayerStats. I simply abstained from changing the method's signature. Should I remove the 'product' parameter?